### DISCUSSION TITLE
Right click to dismiss toast notifications

### DISCUSSION THREAD

**Discussion by hburn7** [ppy/osu#20211] [Ideas] [answered]
Just a simple quality of life improvement, it would be nice to be able to right click -> dismiss the new toast notifications instead of having to find the X or check mark to dismiss them. I believe osu! stable behaves this way anyway.

**Comment by smoogipoo** [comment, marked as answer]
I'd agree with this. Aiming for the :heavy_check_mark: button on the far side of the notification feels really bad and I'm always tempted to click the notification to dismiss it instead.

**Reply by peppy** [reply]
#20215
